pkg/localclusterimport/local_cluster_import_operations.go

```diff
@@ -0,0 +1,179 @@
+package localclusterimport
+
+import (
+	"context"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	hiveext "github.com/openshift/assisted-service/api/hiveextension/v1beta1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+//go:generate mockgen -source=local_cluster_import_operations.go -package=localclusterimport -destination=local_cluster_import_operations_mocks.go
+type ClusterImportOperations interface {
```

Comment:
Why do you need a client wrapper? aren't those regular k8s functions that can be part of `Local_cluster_import`?

Reply:
It's not a very good separation of concerns to be honest. The business action of performing an import vs making operations available in the client are two distinct things.

Moreover it easier for creating a focused set of mocks also.

They could be merged together but I honestly don't think they should be.

Reply:
It's just calling a single function without any logic, this is why i asked.

Reply:
without any logic -> I don't think this is true

At the very least, management of how the responses are marshaled into various objects is being handled here as well as decisions around how the object should be fetched. 

IMO this is a question of where the functions live, if we were to merge the code, we would just be moving the functions from one file to the other. 

I think we would lose the neatness of mocks and it would be harder to maintain.